exporters/prometheus/exporter.go

```diff
@@ -330,9 +330,45 @@ func sanitizeRune(r rune) rune {
 }
 
 var unitSuffixes = map[string]string{
-	"1":  "_ratio",
-	"By": "_bytes",
-	"ms": "_milliseconds",
+	// Time
+	"d":   "_days",
+	"h":   "_hours",
+	"min": "_minutes",
+	"s":   "_seconds",
+	"ms":  "_milliseconds",
+	"us":  "_microseconds",
+	"ns":  "_nanoseconds",
+
+	// Bytes
+	"By":   "_bytes",
+	"KiBy": "_kibibytes",
+	"MiBy": "_mebibytes",
+	"GiBy": "_gibibytes",
+	"TiBy": "_tibibytes",
+	"KBy":  "_kilobytes",
+	"MBy":  "_megabytes",
+	"GBy":  "_gigabytes",
+	"TBy":  "_terabytes",
+	"B":    "_bytes",
+	"KB":   "_kilobytes",
+	"MB":   "_megabytes",
+	"GB":   "_gigabytes",
+	"TB":   "_terabytes",
```

Comment:
"B" is a bel in the UCUM: https://ucum.org/ucum#section-Alphabetic-Index-By-Symbol

```suggestion
```

Reply:
removed